docusaurus.config.js

```diff
@@ -33,8 +33,8 @@ const keywords = [
 const config = {
   title: "Wing",
   tagline: "Maximum cloud, minimum DevOps",
-  url: process.env.DOCUSAURUS_URL ?? `https://${process.env.VERCEL_URL}`,
-  baseUrl: "/",
+  url: process.env.DOCUSAURUS_URL ?? `https://${process.env.VERCEL_URL}/docs/`,
```

Comment:
```suggestion
  url: process.env.DOCUSAURUS_URL ?? `https://${process.env.VERCEL_URL}`,
```

I think this should be sufficient